hugegraph-api/src/main/java/org/apache/hugegraph/api/graph/EdgeAPI.java

```diff
@@ -261,7 +261,7 @@ public String update(@Context GraphManager manager,
 
     @GET
     @Timed
-    @Compress
+    @CompressInterceptor.Compress
```

Comment:
can we search all the `@CompressInterceptor.Compress` and update to `@Compress`?